docs/requirements.txt

```diff
@@ -0,0 +1,8 @@
+PyYAML==6.0.1
```

Comment:
Can we use an `requirements.in` or `pyproject.toml` for these too? Can we dog food here?

Reply:
Yeah... I need to figure out what the input requirements were (this is copied from Ruff).